tests/test_interpreter.py

```diff
@@ -167,3 +186,17 @@ def test_find_python_interpreter_missing_on_path_raises(monkeypatch):
         find_python_interpreter(interpreter)
         assert "Python Launcher" in str(e)
         assert "on your PATH" in str(e)
+
+
+@pytest.mark.skipif(sys.platform.startswith("win"), reason="Looks for python3")
+def test_fetch_missing_python(monkeypatch):
+    major = sys.version_info.major
+
+    # For this test, we want to ask pipx for a version that doesn't exist locally.
+    # Since we test on all supported versions, we can't just pick a specific version to test with.
+    minor = 11 if sys.version_info.minor == 10 else 10
```

Comment:
I'm not quite happy with that. Isn't there some other way to ensure a Python version isn't available?

Reply:
good point, it's been fixed. 

Please have a look and tell me what you think